src/components/ListItem.jsx

```diff
@@ -1,5 +1,43 @@
+import { useEffect, useState } from 'react';
 import './ListItem.css';
+import { updateItem } from '../api';
 
-export function ListItem({ name }) {
-	return <li className="ListItem">{name}</li>;
+export function ListItem({
+	name,
+	itemId,
+	listPath,
+	purchaseDate,
+	nextPurchaseDate,
+}) {
+	let [isPurchased, setIsPurchased] = useState(false);
```

Comment:
Curious why we’re using `let` here instead of `const` for declaring these state variables. Since these variables should not be reassigned, we should consider using `const` to indicate that they are intended to be constant references!